routes/.eslintrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "rules": {
+    "no-param-reassign": "off"
```

Comment:
@kirinrastogi I strongly disagree with this, let it throw the error now, and when we write better code it won't complain (Moving the logic of creating a new keyword in `fetchArticles`)